MonoGame.Framework/Graphics/States/BlendState.OpenGL.cs

```diff
@@ -14,43 +14,78 @@ namespace Microsoft.Xna.Framework.Graphics
 {
     public partial class BlendState
     {
-        internal void PlatformApplyState(GraphicsDevice device)
+        private static bool _currentBlendEnable = false;
+        private static BlendState _currentBlendState = new BlendState();
```

Comment:
Yes, I agree - it should be an internal field / property in `GraphicsDevice`. But since this is specific to OpenGL, probably it should live in `GraphicsDevice.OpenGL.cs`.

(And I might suggest renaming it, since `GraphicsDevice` already has a "current" blend state, even if that's not the actual name of the variable. Perhaps "_deviceBlendState" or something of that nature?)